src/cross_section.F90

```diff
@@ -248,6 +267,12 @@ subroutine calculate_nuclide_xs(i_nuclide, i_sab, E, i_mat, i_nuc_mat, u)
       micro_xs(i_nuclide) % kappa_fission = &
            nuc % reactions(nuc % index_fission(1)) % Q_value * &
            micro_xs(i_nuclide) % fission
+
+      ! Calculate microscopic nuclide delayed nu-fission cross section
+      micro_xs(i_nuclide) % delayed_nu_fission = (ONE - f) * &
+           nuc % delayed_nu_fission(i_grid) + f * &
+           nuc % delayed_nu_fission(i_grid+1)
```

Comment:
There are other tally scores where microscopic cross sections are calculated as needed (just look at the case default in tally.F90:score_general). It's hard to say if it's ok to keep the delayed-nu-fission micro calculation without knowing the impact. For profiling, you have two options -- you can take the lazy approach of just picking a reactor problem and seeing what the difference in performance is (i.e. no profiling really). You can also add profiling flags and use gprof to get an actual profile:
```sh
cmake -Dprofile=on ..
make
...
openmc
gprof openmc_executable gmon.out > profile.txt
```